java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -149,10 +616,11 @@ public String getCode() {
      * @param serverCode code value to search for. Null is acceptable
      * @return Manta error code enum associated with serverCode parameter
      */
+    @SuppressWarnings("Duplicates")
     public static MantaErrorCode valueOfCode(final String serverCode) {
```

Comment:
How do you plan on dealing with codes that are duplicated but not deprecated like `AuthorizationFailed`?

I went through the codes and found the following duplicated:
```
AuthorizationFailed
AuthorizationRequired
AuthorizationSchemeNotAllowed
ContentLengthRequired
ContentMD5Mismatch
DirectoryLimitExceeded
InvalidArgumentError
InvalidHttpAuthenticationToken
InvalidJobState
InvalidMaxContentLength
InvalidQueryStringAuthentication
InvalidResource
LocationRequired
MaxContentLengthExceeded
MultipartUploadInvalidArgument
OperationNotAllowedOnDirectory
OperationNotAllowedOnRootDirectory
SecureTransportRequired
SourceObjectNotFound
```

I might have missed some, so please double check.

Reply:
- Noted, I was working on it as well. 
- I just pushed a [commit](https://github.com/joyent/java-manta/pull/501/commits/a9e095ad0143b975b60c61239431b641c7c90cc9) to tag all the duplicates that are present and aren't **deprecated**. 
- I will come up with a safe implementation soon to handle duplicates.